utils/build.ps1

```diff
@@ -1780,7 +1780,10 @@ function Install-Platform([Platform]$Platform, $Arch) {
   Copy-Directory "$($Arch.SDKInstallRoot)\usr\include\swift\SwiftRemoteMirror" $SDKInstallRoot\usr\include\swift
   Copy-Directory "$($Arch.SDKInstallRoot)\usr\lib\swift\shims" $SDKInstallRoot\usr\lib\swift
   foreach ($Module in ("Block", "dispatch", "os", "_foundation_unicode", "_FoundationCShims")) {
-    Copy-Directory "$($Arch.SDKInstallRoot)\usr\lib\swift\$Module" $SDKInstallRoot\usr\include
+    $ModuleDirectory = "$($Arch.SDKInstallRoot)\usr\lib\swift\$Module"
+    if (Test-Path $ModuleDirectory) {
+      Copy-Directory $ModuleDirectory $SDKInstallRoot\usr\include
+    }
   }
```

Comment:
What about extracting the new directory into a separate clause? That way we can eventually go back to the unconditional form with fewer indentation changes.

Reply:
Sure, do you mean something like

```
foreach ($Module in ("Block", "dispatch", "os", "_foundation_unicode", "_FoundationCShims")) {
  $ModuleDirectory = "$($Arch.SDKInstallRoot)\usr\lib\swift\$Module"
  $DestinationDirectory = "$SDKInstallRoot\usr\include"
  if (Test-Path $ModuleDirectory) {
    Copy-Directory $ModuleDirectory $DestinationDirectory
  }
}
```

Or do you mean extracting `_foundation_unicode`/`_FoundationCShims` into a new `foreach` loop with this check, leaving the existing `foreach` loop as-is so that we can remove the second loop later?

Reply:
Yeap, exactly!

Reply:
👍🏻 addressed with 45521dafeaa3a3ddb3b5b9f307e1e0001629e293